### PR TITLE
First portfolio

### PR DESCRIPTION
**Pull request by kakarrot92**
1. Adding HTML file containing text design of the first section of the portfolio
2. Styling index.html file
3. Adding linter to our pull request

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        </nav>
+      <section class="hero">
+        <h1 class="first">Hey there. I’m Bogdan</h1>
+        <h1 class="second">I’m a software developer</h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </section>
```
Comment (Mihndim2020) [inline index.html:24]: - [ ] As requested by the previous code reviewer, which is also a requirement for this project, please make sure you set images that are backgrounds in the style sheet as background images. You have simply removed the images, now the background is missing. Please kindly fix that. 
***Your implementation***
kakaI

***The design****
sakaD

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  </head>
+  <body>
+    <header>
+        <nav class="navbar">
+          <div class="title"><a>Welcome</a></div>
+          <img src="/images/icon-menu.png" alt="" />
+        </nav>
+      <section class="hero">
+        <h1 class="first">Hey there. I’m Bogdan</h1>
+        <h1 class="second">I’m a software developer</h1>
```
Comment (Mihndim2020) [inline index.html:18]: - [OPTIONAL] Please for accessibility and SEO, your page should have a single `<h1>` tag, please kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @kakarrot92, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there


 # Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @kakarrot92 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Optional Suggestion
- Kindly fix your UI design to look exactly like the Figma template 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**